src/Account.ts

```diff
@@ -12,6 +17,40 @@ export class Account {
     readonly encryptedKeyStore: EncryptedKeystoreV3Json,
   ) { }
 
+  /**
+   * @param {Web3} web3 The web3 instance.
+   * @param password The password required to unlock the account.
+   * @returns {{account: Account; encryptedAccount: PrivateKey}}
```

Comment:
I don't see the availability of `PrivateKey` type.
`{account: EthAccounts.Account, encryptedAccount: EncryptedKeystoreV3Json}`

Reply:
Wrapped into Account class object.